src/View/Helper/TimeHelper.php

```diff
@@ -33,6 +33,21 @@ class TimeHelper extends Helper
     use StringTemplateTrait;
 
     /**
+     * The default format to be used for displaying formatted date strings.
+     *
+     * @var string
+     */
+    public $defaultFormat = null;
```

Comment:
Yes. In `__construct()` and/or `initialize()` and/or via a setter/getter. Just say so.